solutions/ova/kubernetes/firstboot.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+rm -rf /var/tmp/firstboot-execute.log /etc/kubernetes-backup /var/lib/kubelet-backup /var/tmp/firstboot-execute.log
+
+####check pod status####
+function check_hctl_status {
+        sleep 60
+        SECONDS=0
+         # start pods on master nodes
+        kubectl taint nodes $(hostname) node-role.kubernetes.io/master- &>> /var/tmp/firstboot-execute.log
+
+        while [[ SECONDS -lt 1200 ]] ; do
+            if /usr/bin/kubectl exec -it $(/usr/bin/kubectl get pods | awk '/cortx-data-pod/{print $1; exit}') -c cortx-motr-hax -- hctl status &>> /var/tmp/firstboot-execute.log ; then
+                if ! kubectl exec -it $(kubectl get pods | awk '/cortx-data-pod/{print $1; exit}') -c cortx-motr-hax -- hctl status| grep -q -E 'unknown|offline|failed'; then
+                        echo "HCTL pod is up" &>> /var/tmp/firstboot-execute.log
```

Comment:
Do we have any hctl pod in cortx? or it is cortx-data pod?

Reply:
It is cortx-data only. It has alias in background

Reply:
It is about echo statement. how come alias work in echo statement? I guess while echo, you need to give proper pod name, because there is no pod name as HCTL.

Reply:
Yeah that's correct. I misunderstood the your comments. It's done.